salt/beacons/__init__.py

```diff
@@ -129,6 +129,25 @@ def add_beacon(self, name, beacon_data):
 
         return True
 
+    def modify_beacon(self, name, beacon_data):
+        '''
+        Modify a beacon item
```

Comment:
Yeah I think it's ok. We have a fuzzy line between API docs and narrative docs. I wasn't even sure if I should have added the "please fill out the docs more here" comment originally anyway. I think really we'll need a "writing_beacons.rst" page or something (if it doesn't already exist) that the beacon info can be added to, since we're not pulling in this file as part of the doc building process.

I think this is fine as-is for now. Thanks again!